Lib/sqlite3/__init__.py

```diff
@@ -21,3 +21,17 @@
 # 3. This notice may not be removed or altered from any source distribution.
 
 from sqlite3.dbapi2 import *
+
+
+# OptimizedUnicode was deprecated in Python 3.10.  It's scheduled for removal
+# in Python 3.12.
+def __getattr__(name):
+    if name == "OptimizedUnicode":
+        import warnings
+        msg = ("""
+            OptimizedUnicode is obsolete. You can safely remove it from your
+            code, as it defaults to 'str' anyway.
```

Comment:
```suggestion
            OptimizedUnicode is deprecated and will be removed in Python 3.12.
            Since Python 3.3 it has simply been an alias for 'str'.
```
Mind if I change the wording here as well, @vstinner?

Reply:
Do whatever you want, it's your PR :-)

Reply:
All right, I'm done! No more finishing touches :)